Kzl 241 document controller (#86)
kuzzleio/documentation-V2: src/sdk-reference/cpp/1/document/mReplace/index.md

---
layout: sdk.html.hbs
algolia: true
title: mReplace
description:
order: 200
---

# mReplace

> Replaces documents in the persistent data storage.

Reviewer: ```suggestion
Replaces multiple documents.
```

> Returns a [partial error]({{ site_base_path }}api-documentation/errors/#partialerror) (with status 206) if one or more documents can not be replaced.

Reviewer: ```suggestion
Throws a partial error (error code 206) if one or more documents cannot be replaced.
```
